src/module.ts

```diff
@@ -96,7 +96,10 @@ export default defineNuxtModule({
     let tailwindConfig: any = {}
     for (const configPath of configPaths) {
       if (existsSync(configPath)) {
-        const _tailwindConfig = requireModule(configPath, { clearCache: true })
+        let _tailwindConfig: any = {}
+        try {
+          _tailwindConfig = requireModule(configPath, { clearCache: true })
+        } catch (e) {}
```

Comment:
Never do this :) It hides all possible errors. We have `tryRequireModule` to be used instead.

Reply:
![image](https://user-images.githubusercontent.com/1385263/178766538-1b75851a-1910-4a05-a6a7-3b8c092ca09a.png)

Not sure about the difference though 😄

Reply:
Ah good catch. Internal util also ignores non not founds. Do you mind raising an issue to fix?